SignalServiceKit/src/Storage/OWSStorage+Subclass.h

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSStorage.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OWSStorage (Subclass)
```

Comment:
The iOS frameworks use `+Subclass` headers for methods that only subclasses should have access to.  I've followed that pattern here.